client/src/Components/AddMedicine/index.js

```diff
@@ -1,129 +1,123 @@
-	}
+  toggleCheckbox = () => {
+    this.setState({ prescription: !this.state.prescription })
+  }
 
-	AddMedicineInfo = event => {
-		event.preventDefault()
-		const { medName, medCompany, medPrice, prescription } = this.state
-		this.setState({ submitClicked: true })
-		if (medName === '' || medCompany === '' || medPrice === '') {
-			return
-		}
+  AddMedicineInfo = event => {
+    event.preventDefault()
+    const { medName, medCompany, medPrice, prescription } = this.state
+    this.setState({ submitClicked: true })
+    if (medName === "" || medCompany === "" || medPrice === "") {
+      return
+    }
 
-		axios
-			.post('/api/pharmacy/medicine', {
-				medName,
-				medCompany,
-				prescription,
-				medPrice
-			})
-			.then(res => {
-				if (res.data.message == 'true') {
-					this.setState({ msg: 'true' })
-				}
-			})
-	}
+    axios
+      .post("/api/pharmacy/medicine", {
+        medName,
+        medCompany,
+        prescription,
+        medPrice
+      })
+      .then(res => {
+        const { add, message } = res.data
+        if (add && message == "done") {
+          this.setState({ msg: message })
+        } else {
+          this.setState({ msg: message })
+        }
+      })
+  }
 
-	render() {
-		const {
-			medCompany,
-			medName,
-			medPrice,
-			prescription,
-			msg,
-			submitClicked,
-			login
-		} = this.state
-
-		return (
-			<div>
-				<NavBar login={login} {...this.props} />
-				<h3>Medicine Information</h3>
-				<p>Fill The Form To Add A New Medicine</p>
-				<form onSubmit={this.AddMedicineInfo}>
-					<label className="medicineName">Medicine Name</label>
-					<input
-						className="medName"
-						onChange={this.updateInput}
-						value={medName}
-						type="text"
-						name="medName"
-					></input>
-					<label className="priceADD">Price</label>
-					<input
-						className="medPrice"
-						onChange={this.updateInput}
-						value={medPrice}
-						type="text"
-						name="medPrice"
-					></input>
-					<label className="company">Company</label>
-					<input
-						className="medCompany"
-						onChange={this.updateInput}
-						value={medCompany}
-						type="text"
-						name="medCompany"
-					></input>
-					<input
-						className="presc"
-						type="checkbox"
-						name="prescription"
-						onChange={this.toggleCheckbox}
-						value={prescription}
-					/>
-					<label className="checkboxLabel">Needs Prescription</label>
-					<input type="submit" value="Add" className="addInput" />
-					{msg && <p className="addMsg">Your data has been added</p>}
-					{!medCompany && submitClicked && (
-						<p className="addCompany">Please enter a company</p>
-					)}
-					{!medName && submitClicked && (
-						<p className="addName">Please enter a name </p>
-					)}
-					{!medPrice && submitClicked && (
-						<p className="addPrice">Please enter a price</p>
-					)}
-				</form>
-			</div>
-		)
-	}
+  render() {
+    const { submitClicked } = this.state
```

Comment:
you removed the destructing object that used in the last master version and got back the old version